modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -191,24 +196,24 @@ In the On Complete phase, none of these variables (not even the original ones) a
 
 === Scheduling Strategy
 
-A Mule application can contain several batch job definitions. Each definition 
-has its own scheduling strategy. A *Scheduling Strategy* enables you to control 
-how instances of a given batch job are executed: 
+A Mule application can contain several batch job definitions. Each definition
+has its own scheduling strategy. A *Scheduling Strategy* enables you to control
+how instances of a given batch job are executed:
 
-* ORDERED_SQUENTIAL (the default): If several job instances are in an executable
+* `ORDERED_SQUENTIAL` (the default): If several job instances are in an executable
 state at the same time, the instances execute one at a time based on their
 creation timestamp.
-* ROUND_ROBIN: This setting attempts to execute all available instances of a
-batch job using a round-robin algorithm to assign the available resources. 
+* `ROUND_ROBIN`: This setting attempts to execute all available instances of a
+batch job using a round-robin algorithm to assign the available resources.
 +
-The ROUND_ROBIN option is useful when you can guarantee that no job execution can have a 
-side effect on another job execution. Therefore, this option is not a good 
-choice for data synchronization jobs, which can update the same record in two 
-concurrent jobs. Because the order of the job execution is not guaranteed, 
-your result might be a prior version of the data. However, you can safely 
-use this strategy to parallelize the job's execution if your batch job 
-retrieves only new records from a database or lifts individual files 
-from an SFTP server (and you are certain that all records are completely 
+The `ROUND_ROBIN` option is useful when you can guarantee that no job execution can have a
+side effect on another job execution. Therefore, this option is not a good
+choice for data synchronization jobs, which can update the same record in two
+concurrent jobs. Because the order of the job execution is not guaranteed,
+your result might be a prior version of the data. However, you can safely
+use this strategy to parallelize the job's execution if your batch job
+retrieves only new records from a database or lifts individual files
+from an SFTP server (and you are certain that all records are completely
```

Comment:
Suggestion: remove the parenthesis and use a comma instead. (Parentheses can be distracting, so I only use them when necessary.)